app/src/routes/api/v2/area.router.js

```diff
@@ -0,0 +1,305 @@
+            wdpaid: ctx.request.body.wdpaid,
+            userId: userId || ctx.state.loggedUser.id,
+            use: use,
+            iso: iso,
+            datasets,
+            image,
+            tags,
+            status: 'pending',
+            public: public_status 
+        }).save();
+        ctx.body = AreaSerializerV2.serialize(area);
+    }
+
+    static async save(ctx, userId) {
+        await AreaRouterV2.saveArea(ctx, ctx.state.loggedUser.id);
+    }
+
+    static async update(ctx) {
+        const area = await AreaModel.findById(ctx.params.id);
+        const files = ctx.request.body.files;
+        if (ctx.request.body.fields) {
+            ctx.request.body = ctx.request.body.fields
+        }
+        if (ctx.request.body.application || !area.application) {
+            area.application = ctx.request.body.application || 'gfw';
+        }
+        if (ctx.request.body.name) {
+            area.name = ctx.request.body.name;
+        }
+        if (ctx.request.body.geostore) {
+            area.geostore = ctx.request.body.geostore;
+        }
+        if (ctx.request.body.wdpaid) {
+            area.wdpaid = ctx.request.body.wdpaid;
+        }
+        const use = {};
+        if (ctx.request.body.use) {
+            use.id = ctx.request.body.use ? ctx.request.body.use.id : null;
+            use.name = ctx.request.body.use ? ctx.request.body.use.name : null;
+        }
+        area.use = use;
+        const iso = {};
+        if (ctx.request.body.iso) {
+            iso.country = ctx.request.body.iso ? ctx.request.body.iso.country : null;
+            iso.region =  ctx.request.body.iso ? ctx.request.body.iso.region : null;
+        }
+        area.iso = iso;
+        if (ctx.request.body.datasets) {
+            area.datasets = JSON.parse(ctx.request.body.datasets);
+        }
+        if (ctx.request.body.tags) {
+            area.tags = ctx.request.body.tags;
+        }
+        if (ctx.request.body.status) {
+            area.status = JSON.parse(ctx.request.body.status);
+        }
+        if (ctx.request.body.public) {
+            area.public = ctx.request.body.public;
+        }
+        if (files && files.image) {
+            area.image = await s3Service.uploadFile(files.image.path, files.image.name);
+        }
+        if (typeof ctx.request.body.templateId !== 'undefined') {
+            area.templateId = ctx.request.body.templateId;
+        }
+        area.updatedDate = Date.now;
+
+        await area.save();
+        ctx.body = AreaSerializerV2.serialize(area);
+    }
+
+    static async delete(ctx){
+        logger.info(`Deleting area with id ${ctx.params.id}`);
+        const userId = ctx.state.loggedUser.id;
+        let team = null;
+        try {
+            team = await TeamService.getTeamByUserId(userId);
+        } catch (e) {
+            logger.error(e);
+            ctx.throw(500, 'Team retrieval failed.');
+        }
+        if (team && team.areas.includes(ctx.params.id)) {
```

Comment:
DELETE is still dependant on teams. Likely we want a separate delete for FW?

Reply:
yep, we should.